virtualization/windowscontainers/quick-start/quick-start-windows-server.md

```diff
@@ -50,45 +70,66 @@ When PowerShell asks you whether to trust the package source 'DockerDefault', ty
 
 > Tip: If you want to update Docker later:
 >  - Check the installed version with `Get-Package -Name Docker -ProviderName DockerMsftProvider`
 >  - Find the current version with `Find-Package -Name Docker -ProviderName DockerMsftProvider`
 >  - When you're ready, upgrade with `Install-Package -Name Docker -ProviderName DockerMsftProvider -Update -Force`, followed by `Start-Service Docker`
 
-## 2. Install Windows Updates
+## 2. Install Docker EE
+
+This is the Docker Enterprise edition which is the production ready version of the Docker. First you need to uninstall the community edition if you have installed it and want to use this version instead.
 
-Ensure your Windows Server system is up-to-date by running:
+Open an elevated PowerShell session and run the following commands.
 
 ```
-sconfig
+Uninstall-Module -Name DockerMsftProvider -Force
 ```
 
-This shows a text-based configuration menu, where you can choose option 6 to Download and Install Updates:
+Install the Docker EE Provider from the [PowerShell Gallery](https://www.powershellgallery.com/packages/DockerProvider).
 
 ```
-===============================================================================
-                         Server Configuration
-===============================================================================
+Install-Module -Name DockerProvider -Force
+```
 
-1) Domain/Workgroup:                    Workgroup:  WORKGROUP
-2) Computer Name:                       WIN-HEFDK4V68M5
-3) Add Local Administrator
-4) Configure Remote Management          Enabled
+Powershell will ask you to install Nuget provider if you don't have it installed and you have to accept the installation by typing `Y` and hitting enter. Information about the Nuget provider on [PowerShell Gallery](https://www.powershellgallery.com/packages/NuGet).
 
-5) Windows Update Settings:             DownloadOnly
-6) Download and Install Updates
-7) Remote Desktop:                      Disabled
-...
+Now install the latest version of Docker EE.
+```
+Install-Package -Name docker -ProviderName DockerProvider -Force
 ```
+When the installation is complete, reboot the computer.
 
-When prompted, choose option A to download all updates.
+```
+Restart-Computer -Force
+```
+
+After this you are ready to move to section 4.
+
+You can find more information about Docker EE here [Docker Enterpise Edition](https://docs.docker.com/enterprise/).
+
+Detailed Docker EE installation notes [Docker Documentation](https://docs.docker.com/engine/installation/windows/docker-ee/).
+
+> Tip: If you want to update Docker later:
+>  - Check the installed version with `Get-Package -Name Docker -ProviderName DockerProvider`
+>  - Find the current version with `Find-Package -Name Docker -ProviderName DockerProvider`
+>  - When you're ready, upgrade with `Install-Package -Name Docker -ProviderName DockerProvider -Update -Force`, followed by `Start-Service Docker`
+
```

Comment:
Lines 112 - 119 belong in section #1. The upgrade instructions should reference the DockerMsftProvider as in section #1.